src/Cli/Command/RequestCommand.php

```diff
@@ -162,6 +163,9 @@ private function executeFirstRequest($domain, array $alternativeNames)
         // Order
         $domains = array_merge([$domain], $alternativeNames);
         $this->notice(sprintf('Loading the order related to the domains %s ...', implode(', ', $domains)));
+        if (!$this->getRepository()->hasCertificateOrder($domains)) {
+            throw new ProtocolException('You have to ask a challenge first with the command "authorize"');
```

Comment:
We could put the copy pastable command in each error message, would be even easier :)!

Reply:
I refactored code to use a dedicated exception which render a nicer message. 

Not sure to understand what you mean by "each error message"